src/routes/volume/CustomColumn.js

```diff
@@ -99,6 +99,10 @@ const modal = ({
       value: 'dataEngine',
       label: 'Data Engine',
     },
+    {
+      value: 'offlineRebuild',
+      label: 'Offline Replica Rebuild',
+    },
```

Comment:
I would prefer to place it in the custom column, If the setting is important to the user, we can reposition it to the default column.